build.gradle

```diff
@@ -38,17 +38,25 @@ wrapper {
 // https://discuss.gradle.org/t/merge-jacoco-coverage-reports-for-multiproject-setups/12100/6
 task jacocoRootReport(type: JacocoReport) {
   description = 'Merge all coverage reports before submit to SonarQube'
+  dependsOn(subprojects.test)
 
-  executionData project(':spotbugs').file('build/jacoco/test.exec')
+  executionData = files(subprojects.jacocoTestReport.executionData)
   sourceDirectories = files(subprojects.sourceSets.main.allSource.srcDirs)
```

Comment:
Only `main` sourceSet is counted as a target, needs to support others like `gui` in `:spotbugs` subproject.